drivers/winc1500/winc1500.c

```diff
@@ -0,0 +1,416 @@
+/*
+ * Copyright (C) 2017 Bumsik Kim <k.bumsik@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     driver_winc1500
+ * @{
+ *
+ * @file
+ * @brief       The WINC1500 driver (Wi-Fi connectivity only)
+ *
+ * @author      Bumsik Kim <k.bumsik@gmail.com>
+ *
+ * @}
+ */
+
+/* Required for strnlen in string.h, when building with -std=c99 */
+#define _DEFAULT_SOURCE 1
+#include <string.h>
+/* From pkg/winc1500 */
+#include "driver/include/m2m_wifi.h"
+#include "driver/source/m2m_hif.h"
+
+#include "winc1500.h"
+#include "winc1500_internal.h"
+#include "winc1500_netdev.h"
+
+#include "xtimer.h"
+#include "log.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#define START_HANDLER (1)
+#define STOP_HANDLER  (2)
+
+winc1500_t winc1500 = { .state = 0 };
+
+char _handler_stack[WINC1500_STACKSIZE];
+
+msg_t _queue[WINC1500_EVENT_QUEUE_LEN];
+msg_t _mbox_msgs[WINC1500_EVENT_MBOX_LEN];
+
+char _ssid[WINC1500_MAX_SSID_LEN + 1];
+winc1500_ap_t _ap;
+uint8_t *_mac_addr;
+
+static int wait_for_event(msg_t *response, uint16_t event, uint16_t error_event);
+static winc1500_sec_flags_t sec_module2driver(tenuM2mSecType module_sec);
+static tenuM2mSecType sec_driver2module(winc1500_sec_flags_t sec);
+
+/* -------------------------------------------------------------------------- */
+/* Private functions                                                          */
+/* -------------------------------------------------------------------------- */
+#ifdef MODULE_GNRC_NETDEV
+#else
```

Comment:
```C
#ifndef MODULE_GNRC_NETDEV
```

instead?

Reply:
Okay. I will change them. Should I consider using `MODULE_GNRC` here too?

Reply:
Yes, sorry. There were some minutes between this comment and the previous ^^